components/EditButton/index.js

```diff
@@ -0,0 +1,9 @@
+import { EditButtonStyled } from "./EditButton.style";
+
+export default function EditButton({ toggleEditMode }) {
+  return (
+    <EditButtonStyled type="button" onClick={() => toggleEditMode()}>
+      ✍️
+    </EditButtonStyled>
+  );
```

Comment:
if you only have emojis, icons or else in a Button, it needs an additional description to be accessible, an aria-label should do it.